Cargo.toml

```diff
@@ -7,8 +7,12 @@ edition = "2021"
 members = ['verify']
 
 [dependencies]
-wasi = "0.11.0"
-wit-bindgen-guest-rust = { git = "https://github.com/bytecodealliance/wit-bindgen" }
+wasi = { version = "0.11.0", default-features = false }
+
+# TODO: Handle support is temporarily removed in upstream wit-bindgen, so use a
+# temporary fork.
+#wit-bindgen-guest-rust = { git = "https://github.com/bytecodealliance/wit-bindgen" }
```

Comment:
FWIW this is probably going to become more onerous over time. I realize that WASI really wants handles but relying on a fork is going to reduce how useful this all is I think.

Reply:
Yeah. If we need to, we can rewrite all the wit files to pass `i32`'s around for now, but we'll see how things go.